src/config/settings.ts

```diff
@@ -49,6 +49,7 @@ export const defaultSettings = {
   telemetry:       true,
   /** Whether we should check for updates and apply them. */
   updater:        true,
+  debug:          false,
```

Comment:
This is almost a coin flip, but now that we have this, I think it should be true by default, because it's quite common that someone will run into problems early on, and it would be useful to have the debug statements present.

I'm seeing a work item to add debug statements at many important points in the code, such as starting up K8s on lima or WSL.

Reply:
I'd vote the other way — if debug logging is on by default, we might as well not have it.  It should be an option to get _additional_ output for troubleshooting, and never set by default. (See also the terrible mess that is Cloud Foundry logging.)

Reply:
Good point, although I admit I was thinking exactly of that CF logging. They even have unit tests that verify certain logs are being issued.

The issue with having the debug setting off initially is say the user on a first or early use runs into a weird problem and contacts us immediately. We can ask for a log file that might have something helpful in it. If it's a flake, and doesn't happen on a subsequent run after they turn debugging on, we've missed potentially useful info.  But even while writing this I was thinking it's such a long shot. So sure, leave the default at false.